src/runtime/virtcontainers/persist/fs/mockfs_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright Red Hat.
```

Comment:
Should add `2022` here?

Reply:
Not according to the guidelines from Red Hat legal.